tests/client.py

```diff
@@ -54,7 +54,9 @@ def cli_args(add=lambda x: None, accept_unknown=False):
     )
 
     # Some common client binary args are exposed here for ease
-    parser.add_argument("--label", help="Test label name in perf summary output file")
+    parser.add_argument(
```

Comment:
This should default to something like basename(argv[1]) with the suffix stripped I think. And it should maybe just be a suffix for when we run the same thing multiple times with variants of the arguments.